templates/boltzmann.js.tmpl

```diff
@@ -803,31 +805,31 @@ function test ({
   after = require('tap').teardown
 }) {
   const shot = require('@hapi/shot')
-  // <% if postgres %>
+  // {% if postgres == "on" %}
   const database = process.env.TEST_DB_NAME || '<$ service $>_test'
   const postgresClient = new pg.Client({
     connectionString: process.env.PGURL || `postgres://localhost:5432/${database}`
   })
   postgresClient.connect()
-  // <% endif %>
+  // {% endif %}
 
-  // <% if redis %>
+  // {% if redis == "on" %}
   const redisClient = redis.createHandyClient(`redis://localhost:6379/7`)
-  // <% endif %>
+  // {% endif %}
 
-  // <% if postgres or redis %>
+  // {% if postgres or redis %}
   after(() => {
-    // <% if postgres %>
+    // {% if postgres == "on" %}
     postgresClient.end()
-    // <% endif %>
-    // <% if redis %>
+    // {% endif %}
+    // {% if redis == "on" %}
     redisClient.end()
-    // <% endif %>
+    // {% endif %}
   })
-  // <% endif %>
+  // {% endif %}
```

Comment:
It just now clicked for me that we're using `{}`, and I'm happy we're back to it (since we're no longer using helm!)